### PR TITLE
render equations in markdown files

### PR DESCRIPTION
**Pull request by njzjz**
See https://github.blog/changelog/2022-05-19-render-mathematical-expressions-in-markdown/

### CODE REVIEW

**Review by wanghan-iapcm — APPROVED** [review]

File: doc/model/train-energy.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ The construction of the fitting net is give by section `fitting_net`
...
-pref_f(t) = start_pref_f * ( lr(t) / start_lr ) + limit_pref_f * ( 1 - lr(t) / start_lr )
-```
-where `lr(t)` denotes the learning rate at step `t`. `start_pref_f` and `limit_pref_f` specifies the `pref_f` at the start of the training and at the limit of `t -> inf`.
+The loss function $L$ for training energy is given by
+
+$$L = p_e L_e + p_f L_f + p_v L_v$$
+
+where $L_e$, $L_f$, and $L_v$ denote the loss in energy, force and virial, respectively. $p_e$, $p_f$, and $p_v$ give the prefactors of the energy, force and virial losses. The prefectors may not be a constant, rather it changes linearly with the learning rate. Taking the force prefactor for example, at training step $t$, it is given by
+
+$$p_f(t) = p_f^0 \frac{ \alpha(t) }{ \alpha(0) } + p_f^\infty ( 1 - \frac{ \alpha(t) }{ \alpha(0) })$$
```
Comment (wanghan-iapcm) [inline doc/model/train-energy.md:27]: We shall tell the user $p_f^0$ is set by `start_pref_f`, and so on..

Comment (wanghan-iapcm) [reply doc/model/train-energy.md:27]: or we do not use math model here.

File: doc/train/training-advanced.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ The `learning_rate` section in `input.json` is given as follows
...
 * `start_lr` gives the learning rate at the beginning of the training.
 * `stop_lr` gives the learning rate at the end of the training. It should be small enough to ensure that the network parameters satisfactorily converge. 
-* During the training, the learning rate decays exponentially from `start_lr` to `stop_lr` following the formula.
-    ```
-    lr(t) = start_lr * decay_rate ^ ( t / decay_steps )
-    ```
-    where `t` is the training step.
+* During the training, the learning rate decays exponentially from `start_lr` to `stop_lr` following the formula:
+
+$$ \alpha(t) = \alpha_0 \lambda ^ { t / \tau } $$
```
Comment (wanghan-iapcm) [inline doc/train/training-advanced.md:21]: Again we shell tell the user $\alpha_0$ is set by `start_lr`. Or we do not use latex math.

File: doc/model/train-energy.md (hunk 1 above)
Comment (njzjz) [reply doc/model/train-energy.md:27]: I kept both equations.
